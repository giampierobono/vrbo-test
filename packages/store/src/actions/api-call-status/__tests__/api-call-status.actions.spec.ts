import {
  propertiesPollLoadingEndAction,
  propertiesPollLoadingStartAction
} from "../api-call-status.actions";

describe("Api call status actions", () => {
  it("should return correct propertiesPollLoadingStartAction action object", () => {
    expect(propertiesPollLoadingStartAction()).toMatchSnapshot();
  });

  it("should return correcnt propertiesPollLoadingStopAction action object", () => {
    expect(propertiesPollLoadingEndAction()).toMatchSnapshot();
  });
});
