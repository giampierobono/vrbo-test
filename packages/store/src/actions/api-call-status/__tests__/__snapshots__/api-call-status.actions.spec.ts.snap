// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Api call status actions should return correcnt propertiesPollLoadingStopAction action object 1`] = `
Object {
  "type": "[ApiCallStatus] Properties call loading end",
}
`;

exports[`Api call status actions should return correct propertiesPollLoadingStartAction action object 1`] = `
Object {
  "type": "[ApiCallStatus] Properties call loading start",
}
`;
